macOS/Configuration/Extensions/NetworkProtection/VPNProxyExtension.xcconfig

```diff
@@ -53,4 +53,4 @@ SKIP_INSTALL = YES
 LOCALIZATION_PREFERS_STRING_CATALOGS = YES
 SWIFT_EMIT_LOC_STRINGS = YES
 
-LD_RUNPATH_SEARCH_PATHS = @executable_path/../Frameworks @executable_path/../../../../Frameworks
+LD_RUNPATH_SEARCH_PATHS = @executable_path/../Frameworks @executable_path/../../../../../../../../Frameworks
```

Comment:
We should remove VPNProxyExtension entirely from the app, as it's not being used, and will likely not be used again.